src/main/scala/no/ndla/listingapi/model/domain/Card.scala

```diff
@@ -0,0 +1,34 @@
+package no.ndla.listingapi.model.domain
+
+import no.ndla.listingapi.ListingApiProperties
+import org.json4s.FieldSerializer
+import org.json4s.FieldSerializer._
+import org.json4s.native.Serialization._
+import scalikejdbc._
+
+case class ImageCoordinate(col: Long, row: Long)
+case class CoverPhoto(imageApiId: Long, cropStart: ImageCoordinate, cropEnd: ImageCoordinate)
+case class Label(`type`: Option[String], labels: Seq[String])
+case class Card(id: Option[Long],
```

Comment:
Hmm...usikker på om jeg er fan av navnet `Card`. Det gir meg assosiasjoner til hvordan dette presenteres for en bruker som et kort, og gir ikke så mye mening som data fra et Api? Har forsåvidt ingen andre gode forslag her og nå.